### PR TITLE
[RFR] My jobs options

### PR DESCRIPTION
**Pull request by mRoca**

### CODE REVIEW

File: src/SensioLabs/JobBoardBundle/Controller/JobController.php [hunk 1]
```diff
@@ -168,4 +171,29 @@ public function updatePreviewAction(Job $job = null)
...
         );
     }
+
+    /**
+     * @Route("/{country}/{contract}/{slug}/update/status/{status}", name="job_udpate_status")
+     * @ParamConverter("job", class="SensioLabsJobBoardBundle:Job", options={"mapping": {"slug": "slug"}})
+     */
+    public function changeStatusAction(Request $request, Job $job, $status)
+    {
+        if (!$this->get('security.authorization_checker')->isGranted('edit', $job)) {
```
Comment (alOneh) [inline src/SensioLabs/JobBoardBundle/Controller/JobController.php]:
```php
if (!$this->isGranted('edit', $job)) {
    throw ...;
}
```

File: src/SensioLabs/JobBoardBundle/Controller/JobController.php [hunk 2]
```diff
@@ -140,13 +140,16 @@ public function updateAction(Request $request, Job $job)
...
         if ($form->isValid()) {
             $em = $this->getDoctrine()->getManager();
-            $em->persist($job);
             $em->flush();
 
             if ($job->isPublished()) {
                 $this->get('sensiolabs_jobboard.mailer.job')->jobUpdate($job, $oldjob);
             }
 
+            $this->get('session')->getFlashBag()->add('success',
```
Comment (alOneh) [inline src/SensioLabs/JobBoardBundle/Controller/JobController.php]:
```php
$this->addFlash('success', $this->get('translator')->trans('messages.success.job_updated'));
```

File: src/SensioLabs/JobBoardBundle/Controller/JobController.php (hunk 1 above)
Comment (mRoca) [inline src/SensioLabs/JobBoardBundle/Controller/JobController.php]: $this->denyAccessUnlessGranted('edit', $job);

File: src/SensioLabs/JobBoardBundle/Controller/JobController.php (hunk 2 above)
Comment (mRoca) [inline src/SensioLabs/JobBoardBundle/Controller/JobController.php]: Done

**Comment by alOneh** [comment]
:+1:

**Pull request merged**